ci/build_conda_python.sh

```
@@ -4,6 +4,32 @@
+SCCACHE_VERSION="${SCCACHE_VER}" rapids-install-sccache
+
+rapids-mamba-retry update --all -y -n base
+conda install -y conda-build
+
+# install expected Python version
+PYTHON_MAJOR_VERSION=${PYTHON_VERSION%%.*}
+PYTHON_MINOR_VERSION=${PYTHON_VERSION#*.}
+PYTHON_UPPER_BOUND="${PYTHON_MAJOR_VERSION}.$((PYTHON_MINOR_VERSION+1)).0a0"
+PYTHON_MINOR_PADDED=$(printf "%02d" "$PYTHON_MINOR_VERSION")
+PYTHON_VERSION_PADDED="${PYTHON_MAJOR_VERSION}.${PYTHON_MINOR_PADDED}"
+if [[ "$PYTHON_VERSION_PADDED" > "3.12" ]]; then
```

Shellcheck (0.11.0)

[error] Decimals are not supported. Either use integers only, or use bc or awk to compare.

(SC2072)